pandas/core/nanops.py

```diff
@@ -1063,7 +1184,10 @@ def _maybe_null_out(result, axis, mask, min_count=1):
                 # GH12941, use None to auto cast null
                 result[null_mask] = None
     elif result is not tslibs.NaT:
-        null_mask = mask.size - mask.sum()
+        if mask is not None:
+            null_mask = mask.size - mask.sum()
+        else:
+            null_mask = functools.reduce(operator.mul, shape)
```

Comment:
is this line covered in tests? isn't this just ``np.prod(shape)``

Reply:
Yeah, this should be getting hit by `Series(dtype=int).sum()` and the like. I'll confirm once codecov finishes.